aten/src/ATen/native/quantized/cudnn/utils.h

```diff
@@ -203,11 +203,15 @@ at::Tensor getRequantMultiplierTensor(double requant_multiplier, uint8_t num_dim
   return requantize_multiplier_tensor;
 }
 
-uint8_t getAlignment(const at::Tensor &t) {
+uint8_t getAlignment(const Tensor &t) {
   // alignment are in bytes
   uint8_t alignment = 1;
   uintptr_t address = reinterpret_cast<uintptr_t>(t.data_ptr());
-  while (address % alignment == 0 && alignment < 16) alignment *= 2;
+  for (; alignment < 16; alignment *= 2) {
+    if (address % (alignment * 2)) {
+      return alignment;
```

Comment:
was it returning 64 previously? I remember being confused about that

Reply:
It's 64 in the current `Conv_v8.cpp` file, but I'm revisiting this because it could be excessive. Wanted to keep the same max value here as was used previously.